### PR TITLE
Doing some copy changes

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
@alanmoo r?

Fixes copy changes requested in https://github.com/MozillaFoundation/encryption_campaign/issues/254#issuecomment-209521662

File: data/encryptVideos.js [hunk 1]
```diff
@@ -34,11 +34,12 @@ module.exports = [
         "social": "https://d24kjznqej0s8a.cloudfront.net/2016/encryption_campaign/beat5_landingpage",
         "title": "Stand Up For Strong Encryption",
         "description": "We’re seeing more and more governments attempt to undermine encryption. Will you help others understand what encryption is and stand up for strong encryption when it matters most? Sign on to Mozilla's email list and be an encryption champion.",
+        "hybridDescription": "More and more governments are attempting to undermine encryption. You can help by letting us know what country you’d like to receive relevant updates about. This doesn’t sign you up for a new newsletter, but it means you may get an extra email if there’s something specific happening in that country.",
```
Comment (alanmoo) [inline data/encryptVideos.js]: Why couldn't line 36 just be updated?

**Comment by alanmoo** [comment]
LGTM, But put it up on staging or something and make sure @lovegushwa and others see how it's showing across variations (/social vs /hybrid)

**Comment by ScottDowne** [comment]
Yeah it's already checked out that the copy needs to be diff on the two
pages.

**Pull request merged**